### ISSUE TITLE
Create swift3.0 branch

### ISSUE THREAD

**Issue by lfarah** [Codeido/PMAlertController#14]
Hey @Codeido,
I'd love to help **PMAlertController** to be ported to **Swift 3**. Can we create a new branch for that?

**Comment by Codeido** [comment]
Hey @lfarah! Yeah, i think that this is a wonderful idea.
Before creating the branch, you've already used the library with Swift 3?

Let me know ;)
Thanks
-Paolo-

**Comment by lfarah** [comment]
Yes. I just finished porting the example app. Loooooots of changes!

**Comment by Codeido** [comment]
Oh @lfarah i read your comments late.
Anyway, i created the new branch and i've updated the library and the example app to swift 3.
https://github.com/Codeido/PMAlertController/tree/swift3

Let me know if you found something that is not ok.
Thank's very much
Paolo

**Comment by lfarah** [comment]
Awesome! I'll take a look

**Comment by Codeido** [comment]
The master branch actually support swift 2.3. If you need swift 3.0, you have to use this https://github.com/Codeido/PMAlertController/tree/swift3

You can use it via cocoapods:
```
pod 'PMAlertController', :git => https://github.com/Codeido/PMAlertController', :branch => 'swift3'
```

Soon the branch will be merged.

**Comment by lfarah** [comment]
I think it would be nice to merge right now since Xcode 8 was launched. It's always good to give instructions for the people who need the Xcode 7 branch

**Comment by Codeido** [comment]
Hi @lfarah 
Yes sure 👍 
I merged the swift 3 branch inside the master branch, and the new version is now available on cocoapods under 2.0.0 tag.

Check it out: https://github.com/Codeido/PMAlertController/releases/tag/2.0.0

Let me know if you want to change something inside the readme file or inside the release log.
